src/Controller/StatisticsController.php

```diff
@@ -67,16 +85,78 @@ public function generalStatistics(
                 "stack" => 'Hommes'
             ])
             ->labels('seasonName');
-        $totalChart = $totalBuilder->buildChart('total-chart', Chart::BAR);
-        $totalChart->pushOptions([
-            'scales' => ([
-                'xAxes' => ([
-                    'stacked' => 'true'
-                ]),
-                'yAxes' => ([
-                    'stacked' => 'true'
-                ])
-            ])
+        $totalChart = $totalBuilder->buildChart('total-chart', MChart::BAR);
+
+        $totalLicences = $subscriptionRepository->totalLicencesPerSeason();
+        $seasonNames = [];
+        foreach ($seasons as $season) {
+            $seasonNames[] = $season->getName();
+        }
+
+        $data = [
```

Comment:
je comprends pas ce tableau avec tout à zéro, enfin plutot le nombre de zéro il vient d'où ?

Reply:
autant de 0 que de type de licence ^^ itérer des 0 avec un count de l'array $licences était pas terrible et me rajoutait des lignes de codes donc j'ai mis à la main...

Reply:
et ça m'évite de faire un else = 0 pour toutes mes licences dans la boucle si la licence n'est pas trouvée sur la saison (donc qu'il n'y a pas ce type de licence sur cette saison et que le total est à 0)

Reply:
mais quand tu vas ajouter une nouvelle saison, ça ne fonctionnera plus si tu fixes à la main le nombre de zéro non ?

Reply:
ok je l'ai rajouté dans la boucle sur mes saisons pour que ça me mettent autant de 0 que de saisons :)